spec/tokenized-buffer-spec.coffee

```diff
@@ -580,7 +572,7 @@ describe "TokenizedBuffer", ->
 
     describe "when the selector matches a run of multiple tokens at the position", ->
       it "returns the range covered by all contigous tokens (within a single line)", ->
-        expect(tokenizedBuffer.bufferRangeForScopeAtPosition('.function', [1, 18])).toEqual [[1, 6], [1, 28]]
+        expect(tokenizedBuffer.bufferRangeForScopeAtPosition('.meta.function', [1, 18])).toEqual [[1, 6], [1, 28]]
```

Comment:
Merge commit 0cd1f110b581e4c29fa4cec432cdbb93ecba2409 reproduces bug, and it's parent d75d202d337142e206e500c10fed34151b4cc750 does not reproduce.